android/src/main/java/com/klarna/inapp/sdk/PaymentViewWrapper.java

```diff
@@ -72,20 +79,23 @@ public void requestLayout() {
         innerWebView.evaluateJavascript("document.body.scrollHeight", new ValueCallback<String>() {
             @Override
             public void onReceiveValue(String value) {
-                if(value == null || value.equals("null") || value.equals("undefined")){
+                if (value == null || value.equals("null") || value.equals("undefined")) {
                     return;
                 }
                 try {
                     final float contentHeight = Float.parseFloat(value);
                     final float scaledHeight = contentHeight * displayDensity;
+                    if (scaledHeight > 0 && !attachedResizeObserver.get()) {
+                        attachResizeObserver(innerWebView);
+                    }
```

Comment:
On the issue #41 its reported that `scaledHeight` is set to 0 due to parsed value, we need to utilize this receiver in that case too.

Reply:
Yes, based on this we can remove the check on calculated height value. It would always try to attach the observer until we receive a resized() callback.